pkgs/development/python-modules/twisted/default.nix

```diff
@@ -128,6 +121,7 @@ buildPythonPackage rec {
   checkInputs = [
     git
     glibcLocales
+    hypothesis
```

Comment:
That explains why some tests failed when I tried to update and couldn't find an error log.